classes/Datasets/Transforms.sc

```diff
@@ -0,0 +1,122 @@
+/*****************************************
+Data Transforms
+(C) 2018 Jonathan Reus
+
+Tools for analyzing and transforming datasets: scaling, normalization, standardization, PCA, etc..
+
+
+******************************************/
+
+// TO NORMALIZE:
+// find min / max of features
+// normalized_value = (val - min) / (max - min)
+
+
+/*
+T for Transform
```

Comment:
often, the need to explain is an indication that a name could be improved. If this class is not used dense repetiton, like classes like `Array` or `Event`, you could just as well write it out.

`TransformNormalizer`

This would be better because `T` may mean a lot of different things (think of task  in`Tdef`), and `Normalizer` is a UGen.

Also possible is to make the verb the name of a class:

`NormalizeTransform`

Both I find fine.